crates/rolldown/tests/rolldown/function/file/js_css/artifacts.snap

```diff
@@ -6,6 +6,14 @@ snapshot_kind: text
 
 ## dist/out.css
 
+```css
+body {
+  background-color: #f0f0f0;
+}
+
+```
+## out.css
+
```

Comment:
Here generated a js file and a css file, but it is enabled `file`, it should be throw a error `Two output files share the same path but have different contents`. [esbuild playground](https://hyrious.me/esbuild-repl/?version=0.24.2&b=e%00entry.js%00import+%27.%2Fa.css%27%0A%0Aconsole.log%281%29&b=%00a.css%00div+%7B%0A++color%3A+%23666%3B%0A%7D&o=--bundle+--outfile%3D%22dist%2Findex.css%22)

Reply:
https://hyrious.me/esbuild-repl/?version=0.24.2&b=e%00entry.js%00import+%27.%2Fa.css%27%0A%0Aconsole.log%281%29&b=%00a.css%00div+%7B%0A++color%3A+%23666%3B%0A%7D&o=--bundle+--outfile%3D%22dist%2Findex.js%22

It depends

Reply:
Interesting behavior...